src/olympia/accounts/views.py

```diff
@@ -211,15 +217,18 @@ def inner(self, request):
 
 def add_api_token_to_response(response, user, set_cookie=True):
     # Generate API token and add it to the json response.
-    payload = jwt_api_settings.JWT_PAYLOAD_HANDLER(user)
-    token = jwt_api_settings.JWT_ENCODE_HANDLER(payload)
+    data = {
+        'secret': user.get_session_auth_hash(),
```

Comment:
Maybe `token_id`? I just think calling it `secret` could be misleading since it will be publicly accessible. Doesn't really matter though.